docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/04 - Subprojects.md

```diff
@@ -0,0 +1,45 @@
+# Subprojects
+
+## Overview
+The **Subproject** plays an important role in determining the instrument battery that will be assigned to a candidate at a timepoint. **Subprojects** are defined in the `subproject` table of the database.
+
+## Adding Subproject Options
+
+### Front End (Recommended)
+Subprojects are defined in the Configuration module, which can be found in LORIS under the **Admin** menu tab.  Click on _To configure study subprojects click here_ link at the top of the page. Refer to the help section of the module for further instructions on how to add or modify subprojects.
+   
+### SQL
+Subprojects can be added directly in SQL using the following command.
+
+```sql
+INSERT INTO subproject (title) VALUES('SCI');
+```
+
+
+### API
+ _not yet available. See [API documentation](../../../API/) for latest additions_
+ 
+
+## Interaction With LORIS
+
+### Projects
+**Subprojects** must be associated to Projects in order to be able to create timepoints for candidates. This association should be defined directly on the front end through the configuration module.
```

Comment:
there's ambiguity in -> `**Subprojects** must be associated to Projects`
- [ ] Important to clarify here (because there's no other place where it's more important to mention this) -- can one subproject be associated to many projects?  Should it? 

- [ ]  How do you delete an association between a subproject and a project? 
- [ ]  How do you update/change an association between a subproject and a project?

Reply:
I'm not sure i understand, it's all in the front end... like if you access the module you can add or remove these associations like any other multiselect

Reply:
Ok, fair enough about the last 2 points.  
Can you address the first one tho? 

Also, there's no way to see in the front-end how *many* projects a given Subproject is already affiliated to.  
I'd suggest we could write that back-end query snippet here.

Reply:
If it's available in the list then yeah !! otherwise it shouldnt be in the list in the first place

Reply:
hey, i think you're talking about the projects page.  From the subprojects page you can't see all the projects the given subproject is or can be affiliated to.

Reply:
Okai... So im not sure why i have to talk about it here or give a back end query. Its not like im giving a back end query for how many battery rows are using a specific suproject. Theres no bound on the number of tables using subproject. Im already mentioning that the information is available through the project configuration page. More than that developers can figure their own sql queries and/or someone should add it to the front end and then complete this doc for their changes

Reply:
ok i'll suggest another edit to show what i mean.